internal/provider/provider_test.go

```diff
@@ -172,7 +172,10 @@ func testAccPreCheck(t *testing.T) {
 		}
 	}
 	confResp := configureProvider(t, Provider)
-	assert.Equal(t, confResp.Diagnostics.HasError(), false)
+	errs := confResp.Diagnostics.Errors()
```

Comment:
Very simple really, `assert.Equal(t, confResp.Diagnostics.HasError(), false)`, on error, only displays that there is an error

My change is an identical assertion, but on error we actually show what the errors are.

I made this change whilst debugging a test. However, since it's a marginal improvement I thought it's worth keeping, hopefully it'll save someone some time

Reply:
Good point!

I'll use `assert.Equal` to avoid the extra import, and to keep the assertions the same